src/core/core.adapters.js

```diff
@@ -26,8 +26,9 @@ function abstract() {
 
 export class DateAdapter {
 
-  constructor(options) {
+  constructor(options, chartOptions) {
```

Comment:
Maby `localeOptions` might be a better name, chartOptions feels a bit misleading, get the feeling like all the chart opts are passed with that name but that is not the case